src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -253,6 +260,14 @@ public abstract class Controller
     static final boolean ALLOW_UNSAFE_AGGRESSIVE_SSTABLE_EXPIRATION = Boolean.parseBoolean(System.getProperty(ALLOW_UNSAFE_AGGRESSIVE_SSTABLE_EXPIRATION_PROPERTY));
     static final boolean DEFAULT_ALLOW_UNSAFE_AGGRESSIVE_SSTABLE_EXPIRATION = false;
 
+    /**
+     * This property allows seperate defaults for vector and non-vector tables.  If this property is set to true
+     * and the table has a {@link VectorType}, the "vector" defaults are used over the regular defaults.  For instance,
+     * "-Dunified_compaction.vector_sstable_growth" will be used over "-Dunified_compaction.sstable_growth".
+     */
+    static final String OVERRIDE_UCS_CONFIG_FOR_VECTOR_TABLES_PROPERTY = PREFIX + "override_ucs_config_for_vector_tables";
+    static final boolean OVERRIDE_UCS_CONFIG_FOR_VECTOR_TABLES = Boolean.parseBoolean(System.getProperty(OVERRIDE_UCS_CONFIG_FOR_VECTOR_TABLES_PROPERTY, "false"));
```

Comment:
good to have this disabled by default here, we can enabled it in CNDB services.

I don't know for Cassandra (HDC) Vector users what is the best ?
But we are not in a hurry to set the default to true now, we can have a conversation with the HDC team (cc @szymon-miezal )